client/e2e/pages/AdminPurchasePage.ts

```diff
@@ -0,0 +1,206 @@
+/* eslint-disable require-jsdoc */
+import test, {ElementHandle, type Locator, type Page} from '@playwright/test';
+import {CustomerInfo} from '../testData/CustomerInfo';
+import {CreditCardInfo} from '../testData/CreditCard';
+
+export class AdminPurchasePage {
+  readonly page: Page;
+
+  readonly loadingScreen: Locator;
+  readonly purchaseSuccessfulScreen: Locator;
+  readonly purchaseTicketButton: Locator;
+
+  readonly eventDropdown: Locator;
+  readonly eventTimeDropdown: Locator;
+  readonly ticketTypeDropdown: Locator;
+  readonly checkoutButton: Locator;
+
+  readonly firstName: Locator;
+  readonly lastName: Locator;
+  readonly phone: Locator;
+  readonly email: Locator;
```

Comment:
It looks like the only issue is updating the page to handle the `Confirm Email` field.

Reply:
Fixed